lib/standalone.mjs

```diff
@@ -60,9 +60,7 @@ export default async function standalone({
 }) {
 	const startTime = Date.now();
 
-	const isValidCode = typeof code === 'string';
-
-	if ((!files && !isValidCode) || (files && (code || isValidCode))) {
+	if ((!files && !isString(code)) || (files && isString(code))) {
 		return Promise.reject(
 			new Error('You must pass stylelint a `files` glob or a `code` string, though not both'),
```

Comment:
nit

It's missing `--stdin`.
i.e. from the user perspective `code` usage is an internal implementation detail
see https://github.com/stylelint/stylelint/blob/main/lib/cli.mjs#L539

Reply:
No, `code` is a part of the Node.js API.

Reply:
Um, I don't think it's necessary to mention `--stdin` here because this module has an interest in Node.js API, not CLI.